src/Uno.UI/ts/WindowManager.ts

```diff
@@ -1270,8 +1270,7 @@ namespace Uno.UI {
 			const resultWidth = offsetWidth ? offsetWidth : element.clientWidth;
 			const resultHeight = offsetHeight ? offsetHeight : element.clientHeight;
 
-			// +1 is added to take rounding/flooring into account
-			return [resultWidth + 1, resultHeight];
+			return [resultWidth, resultHeight];
```

Comment:
The original reason for the presence of this was to avoid rounding errors with the way text measure behaves, vs how HTML actually lays out text. In some cases, wrapping would occur at the HTML level because the available size of the parent container would not fit the text.

I would not be surprised if screenshots would fail.

@carldebilly may also have comments on this particular change.